lib/wibox/widget/textbox.lua

```diff
@@ -73,13 +73,10 @@ end
 -- @treturn number The preferred width.
 -- @treturn number The preferred height.
 function textbox:get_preferred_size(s)
-    local dpi
-    if s then
-        dpi = screen[s].dpi
-    else
+    if not s then
```

Comment:
What is with `:fit` and `:draw`? This commits make the "non-widget" functions use a different DPI value than the actual drawing does.